test/e2e/util.go

```diff
@@ -780,6 +781,21 @@ func removeTaintsFromAllNodes(ctx *context, taints []v1.Taint) error {
 	return nil
 }
 
+func getAllNodes(ctx *context) []string {
+	nodeNames := make([]string, 0)
+
+	nodes, err := ctx.kubeclient.CoreV1().Nodes().List(metav1.ListOptions{})
+	Expect(err).NotTo(HaveOccurred())
+
+	for _, node := range nodes.Items {
+		if strings.Contains(node.Name, "master") {
```

Comment:
check `node.Spec.Taints` instead of string. And we can not make sure node name contains "master" is the master.